app.Dockerfile

```diff
@@ -10,17 +10,26 @@ RUN echo 'deb http://apt.postgresql.org/pub/repos/apt/ stretch-pgdg main' >  /et
     && curl -s https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - \
     && apt-get update && apt-get install -y postgresql-client
 
-COPY deploy/requirements.txt /deploy/requirements.txt
-RUN pip install --no-cache-dir --disable-pip-version-check --requirement /deploy/requirements.txt
+# Install nodejs dependencies
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
```

Comment:
This is great for now, but it'd be cool to think about how we want this to work in a deployed environment. Might be worth trying to make a more lightweight version where we don't need node running so we can just use pre-built webpack bundles to handle this stuff. (definitely a separate issue)

Reply:
Worth pondering how to make is more lightweight. We could build + compile node on the CircleCI containers rather than on the joplin container itself. 

Our current deployed environments already do install node. The change I made to the dockerfile just installs node on our local containers as well. 

That was actually a misfired attempt to do something else. I thought that sass had to be rebuilt on the joplin_app container rather than the joplin_asset container.